src/scss/main.scss

```diff
@@ -259,3 +259,7 @@ textarea {
     }
   }
 }
+
+.relative {
+  position: relative;
+}
```

Comment:
There is no need to create a separate class for a single property. Please delete